Add language reporter block to translate extension

picklesrus opened the pull request:
### Resolves
Progress toward https://github.com/LLK/scratch-vm/issues/1029

Adding the reporter block for viewer language. The block reports the human readable language name since that is easier to understand and the translate block accepts language names dropped on top of the menu.

Also adds messages for the text in the blocks

Removes the icons from the blocks. These will be replaced in a subsequent PR.

picklesrus committed:
Add language reporter block to translate extension

ericrosenbaum commented:
I can't figure out how to push directly to this PR- here's what I was playing with. It updates the viewer language code, and converts it to the language name in that language:

```
    getViewerLanguage () {
        this._viewerLanguageCode = this.getViewerLanguageCode();
        const names = languageNames.menuMap[this._viewerLanguageCode];
        const langNameObj = names.find(obj => obj.code === this._viewerLanguageCode);
        let langName = this._viewerLanguageCode;
        if (langNameObj) {
            langName = langNameObj.name;
        }
        return langName;
    }
```

picklesrus committed:
- Update the reporter to return the human readable language.
- Remove the non-final icons and replace with a TODO.
- Add messages for the text in the blocks.

picklesrus committed:
remove stray space

picklesrus commented:
@ericrosenbaum Added that bit of code and added messages in block text. I also went ahead and removed the icons since I was here.  Updated the PR description to reflect this.  Should be ready for another look.

ericrosenbaum reviewed (approved):
Looks great!
